ha/util/conf_store.py

```diff
@@ -29,6 +30,17 @@
 from cortx.utils.log import Log
 from ha.core.error import ClusterCardinalityError
 
+
+# Confstore keys
+class GconfKeys(enum.Enum):
+    CVG_NAME = "node>(node_id)>storage>cvg[(cvg_index)]>name"
+    CVG_COUNT = "node>(node_id)>storage>num_cvg"
+    DATA_COUNT = "node>(node_id)>storage>cvg[(cvg_index)]>devices>num_data"
+    METADATA_COUNT = "node>(node_id)>storage>cvg[(cvg_index)]>devices>num_metadata"
+    DATA_DISK = "node>(node_id)>storage>cvg[(cvg_index)]>devices>data[(d_index)]"
+    METADATA_DISK = "node>(node_id)>storage>cvg[(cvg_index)]>devices>metadata[(m_index)]"
```

Comment:
Instead of replacing the str, we can use placeholder. This will avoid accidently replacing matching string.
Ex:
```
CVG_NAME = f"node>{node_id}>storage>cvg[{cvg_index}]>name"

CVG_NAME.format(node_id=1, cvg_index=2)
```

We need not to even convert any data to str.

Reply:
No, we can't use formatting here.
Without specifying node_id & cvg_index we can't use formatting here _f"node>{node_id}>storage>cvg[{cvg_index}]>name"_

Reply:
My bad, avoid f"". But still that will work, @akash2144 
```
>>>
>>> CVG_NAME = "node>{node_id}>storage>cvg[{cvg_index}]>name"
>>> CVG_NAME.format(node_id=1, cvg_index=2)
'node>1>storage>cvg[2]>name'
>>>
```

Reply:
Done, Thanks!